Formula/safe.rb

```diff
@@ -0,0 +1,40 @@
+class Safe < Formula
+  desc "Vault CLI"
+  homepage "https://github.com/starkandwayne/safe"
+  url "https://github.com/starkandwayne/safe/archive/v0.8.1.tar.gz"
+  sha256 "cc766996cfff2109c27f98be0430fbc0d16076cb6772de6b883307e9d95b6aae"
+
+  depends_on "go" => :build
+  depends_on "gox" => :build
+  depends_on "vault"
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["RELEASE_ROOT"] = buildpath/"destdir"
+    ENV["TARGETS"] = "darwin/amd64"
+    ENV["VERSION"] = version
+
+    (buildpath/"src/github.com/starkandwayne/safe").install buildpath.children
+
```

Comment:
This empty line is probably redundant.

Reply:
line 18?  sure, i can remove that.

Reply:
Aye. It's a minor thing but brew seems to have become a bit tighter on whitespace lately so since there's other things to update/change, why not 🤷‍♂️.